### ISSUE TITLE
Radarr Synology v2 > v3

### ISSUE THREAD

**Issue by DejaVu** [Radarr/Radarr#4927]
An update on the latest version on Radar v0.2.0.1504 explains that an internal upgrade can be made to aphrodite. 
This worked at around the time it was added (July 3rd). A change in v3 has since broken it. 

Update now goes through the motions, downloads, moves the files to the correct location, but then fails to start. Can this be revisited? 

Radarr Version: v0.2.0.1504
Mono Version: v5.2.0.1.34
Debug Logs: - https://pastebin.com/SD5B7WiS - https://pastebin.com/YLhWM8hn - https://pastebin.com/tYB0HZLG

Manually restarting Radarr after the update fails. It shuts down and doesn't load again until you reinstall from scratch.

**Comment by Qstick** [comment]
@ta264 this related to #4890 or something different?

**Comment by ta264** [comment]
Ah yeah probably

**Comment by Qstick** [comment]
Can you test latest Aphrodite? 3544

**Comment by DejaVu** [comment]
I can report back this fix has worked perfectly for me. Back on v3! 
Thanks for such a speedy fix. Much appreciated!

**Comment by bakerboy448** [comment]
run it as netcore rather than as mono

**Comment by DejaVu** [comment]
No option to change that from within Synology - without knowledge of where/how it's started (using SSH). 
I have found a script, but it's above my skills to work out what is what and how to edit it. I know it's simply changing "mono --debug Radarr.exe" to simply "Radarr", but that is awkward to work out for someone like me. Tried for 3 days! lol

The start-up scripts are buried deep. I think those of us on this system may need to wait for a new release from SynoCommunity to start it correctly. I've been delving into compiling a v3 version, but I'm not a script writer unfortunately.
